modules/mysql/main.tf

```diff
@@ -19,8 +19,8 @@ locals {
   ip_configuration_enabled = "${length(keys(var.ip_configuration)) > 0 ? true : false}"
 
   ip_configurations = {
-    enabled  = "${list(var.ip_configuration)}"
-    disabled = "${list()}"
+    enabled  = "${var.ip_configuration}"
+    disabled = "${map()}"
```

Comment:
Assuming this change is correct, we also need to update postgre's one: https://github.com/GoogleCloudPlatform/terraform-google-sql-db/blob/02a3a5d1514cd082ab1afc3af6f19192cadeb76c/modules/postgresql/main.tf#L22

Reply:
That change is correct.

PSQL has been updated as well.